corgidrp/calibrate_kgain.py

```diff
@@ -281,7 +277,7 @@ def calibrate_kgain(dataset_kgain,
                     n_cal=10, n_mean=30, min_val=800, max_val=3000, binwidth=68,
                     make_plot=True,plot_outdir='figures', show_plot=False,
                     logspace_start=-1, logspace_stop=4, logspace_num=200,
-                    verbose=False, detector_regions=None):
+                    verbose=False, detector_regions=None, kgain_params=kgain_params_default):
```

Comment:
the python "best practices" on this is to set `kgain_params=None` and later in the code, `if kgain_params is None: kgain_params = default`. This is to avoid bugs that can occur when setting python objects (which are mutable) as default arguments.